doc/classes/ProjectSettings.xml

```diff
@@ -305,6 +311,8 @@
 		<member name="application/run/low_processor_mode_sleep_usec" type="int" setter="" getter="" default="6900">
 			Amount of sleeping between frames when the low-processor usage mode is enabled (in microseconds). Higher values will result in lower CPU usage.
 		</member>
+		<member name="application/run/main_loop_type" type="String" setter="" getter="" default="&quot;SceneTree&quot;">
+		</member>
```

Comment:
```suggestion
		<member name="application/run/main_loop_type" type="String" setter="" getter="" default="&quot;SceneTree&quot;">
			The name of the type implementing the engine's main loop.
		</member>
```

Reply:
Thank you were uncertain how to phrase this one